packages/styles/scss/components/checkbox/_checkbox.scss

```diff
@@ -180,6 +180,31 @@
     background-color: $icon-disabled;
   }
 
+  //----------------------------------------------
+  // Read-only
+  // ---------------------------------------------
+  // should this be a token of it's own?
+  $border-readonly: $icon-disabled;
```

Comment:
I'd just pull this out for now unless there's clear indication from design that we're going to add tokens for this.

Reply:
Confirmation to pull this out for now - https://github.com/carbon-design-system/carbon/issues/12243#issuecomment-1293607905:

> I personally don't think we need read-only tokens. I think using a mix of regular and disabled color tokens in this context is acceptable. If the style ever diverges, we can make a specific one. Its only used in a few read-only components. We could make a whole suite of read-only tokens but I think that is overkill.